tests/VObject/VCardBirthdayTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Sabre\VObject;
+
+class VCardBirthdayTest extends TestCase {
+
+    function testValidBirthday() {
+
+        $input = <<<VCF
+BEGIN:VCARD
+VERSION:3.0
+N:Gump;Forrest;;Mr.
+FN:Forrest Gump
+BDAY:19850407
```

Comment:
Hey that's my birthday! Are you calling me forest gump? :P

Regardless... you need to add a test for BDAY values without a year and without a month.